common/src/main/java/org/jboss/gm/common/groovy/BaseScript.java

```diff
@@ -49,9 +49,10 @@
      */
     @Override
     public Project getProject() {
-        if (stage == InvocationStage.FIRST) {
-            logger.error("getProject unsupported for InvocationStage FIRST");
-            throw new ManipulationUncheckedException("getProject is not supported for Groovy in initial stage.");
+        if (stage == InvocationStage.PREPARSE || stage == InvocationStage.FIRST) {
+            logger.error("getProject unsupported for InvocationStage {}", stage.name());
+            throw new ManipulationUncheckedException(
```

Comment:
I slightly changed the exception message to make this one and the one below consistent.

Reply:
Ok